client/signup/steps/site-type/index.jsx

```diff
@@ -42,12 +47,43 @@ class SiteType extends Component {
 		}
 	};
 
+	renderImportButton() {
+		if (
+			! isEnabled( 'signup/import-flow' ) ||
+			'show' !== abtest( 'showImportFlowInSiteTypeStep' )
+		) {
+			return null;
+		}
+
+		return (
+			<div className="site-type__import-button">
+				<Button borderless onClick={ this.props.goToNextStep.bind( this, 'import' ) }>
```

Comment:
Could we do something like this to avoid having to bind in the component please?

```js
handleImportFlowClick = () => this.props.goToNextFlow( 'import' );
...
<Button borderless onClick={ this.handleImportFlowClick }>
```